pkg/apis/serving/v1beta1/predictor_xgboost.go

```diff
@@ -26,7 +26,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// SKLearnSpec defines arguments for configuring SKLearn model serving.
+// XGBoostSpec defines arguments for configuring XGBoost model serving.
```

Comment:
thanks for spotting this! can you help fix [here](https://github.com/kubeflow/kfserving/blob/7452690e7188755c2b45d785ff5371b4b1451690/pkg/apis/serving/v1beta1/swagger.json#L3778) too?
you can run `hack/update-openapigen.sh` to update that.

Reply:
Sure, will fix it.

Reply:
Fixed @yuzisun